config/initializers/new_framework_defaults_6_1.rb

```diff
@@ -0,0 +1,67 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
I believe the goal of this file is to uncomment each setting one at a time to make sure your app still works before flipping the main switch in the application.rb over to 6.0.

Once everything is working, you can delete this file.